playground/backend/internal/environment/environment_service.go

```diff
@@ -40,6 +40,10 @@ const (
 	cacheKeyExpirationTimeKey     = "KEY_EXPIRATION_TIME"
 	pipelineExecuteTimeoutKey     = "PIPELINE_EXPIRATION_TIMEOUT"
 	protocolTypeKey               = "PROTOCOL_TYPE"
+	launchSiteKey                 = "LAUNCH_SITE"
+	googleProjectIdKey            = "GOOGLE_CLOUD_PROJECT"
+	defaultLaunchSite             = "local"
+	defaultGoogleProjectId        = ""
```

Comment:
do we need a default project id if it is empty?

Reply:
Removed this const.